src/page/renderer/field/inputGenerator.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export default (type, baseProps) => props => <input { ...props } { ...baseProps } type={ type } />;
```

Comment:
why it has a special generator in a new file?
can't it be just:
```javascript
case 'text':
   return props => <input { ...props } { ...baseProps } type={ type } />;
```
in field.js ?

Reply:
resolved on private conversation